packages/cli-template-monorepo-subgraph/apps/web-app/package.json

```diff
@@ -1,25 +1,29 @@
 {
     "name": "monorepo-subgraph-web-app",
-    "version": "1.0.0",
+    "version": "0.1.0",
     "private": true,
     "scripts": {
         "dev": "next dev",
         "build": "next build",
-        "export": "next export",
-        "start": "next start"
+        "start": "next start",
+        "lint": "next lint"
     },
     "dependencies": {
-        "@next/font": "13.0.3",
-        "@semaphore-protocol/core": "4.0.0-alpha.8",
-        "@semaphore-protocol/data": "4.0.0-alpha.8",
-        "@types/react": "18.0.25",
-        "@types/react-dom": "18.0.8",
-        "dotenv": "^16.0.3",
-        "ethers": "^6.4.0",
-        "next": "13.0.3",
+        "@semaphore-protocol/core": "^4.0.0-alpha.8",
+        "@semaphore-protocol/data": "^4.0.0-alpha.8",
```

Comment:
I get it for the contracts, but what would be the issue with the JS libraries if the JS libraries are following the semantic versioning?

Reply:
Just having more control of the actual Semaphore code the app is using. With `^` it could use `4.1.0` or `4.12.0`. Minor versions could have important changes on JS libraries.